storybooknative/src/InputField/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow strict
+
+import * as React from "react";
+import { TextInput } from "react-native"; // eslint-disable-line no-restricted-imports
+
+import Text from "../Text";
+import styled from "../styled";
+
+type Props = {|
+  +placeholder?: string,
```

Comment:
The placeholder should also have been left to be of type Text I believe.

Reply:
In docs, placeholder should be of type string. I thought that I would leave it like that for now, and change it when I make placeholder as a separate Text component, if it comes to that.